### DISCUSSION TITLE
Adding emoji picker to the Radicale server

### DISCUSSION THREAD

**Discussion by GreenGusher** [Kozea/Radicale#1367] [Ideas]
Hi,
I would like to brighten up my radicale experience for myself and my family by incorporating an emoji picker into the server titles and text boxes.  I guess that begs the question of a broader scope of things like Apache or Nginx incorporating emoji pickers.  Or even http addreses lol.
Radicale has been working great for me.
Nathan

**Comment by alqu** [comment]
Seems to be loosely connected with #1232 ... maybe we can build on that update and add the emoji-picker afterwards
